src/Implementation/WarmStorage/WarmStorage/WarmStorageCoordinator.cs

```diff
@@ -56,8 +56,7 @@ private WarmStorageCoordinator(EventProcessorHost host)
             {
                 MaxBatchSize = configuration.MaxBatchSize,
                 PrefetchCount = configuration.PreFetchCount,
-                ReceiveTimeOut = configuration.ReceiveTimeout,
-                InvokeProcessorAfterReceiveTimeout = true
+                ReceiveTimeOut = configuration.ReceiveTimeout
```

Comment:
I tried removing InvokeProcessorAfterReceiveTimeout (the default is false) and EH still called me  with null events on timeout. That is why I added the check anyways.